docs/csharp/language-reference/compiler-options/listed-alphabetically.md

```diff
@@ -9,69 +9,73 @@ helpviewer_keywords:
-  
-|Option|Purpose|  
-|------------|-------------|  
-|[@](response-file-compiler-option.md)|Reads a response file for more options.|  
-|[-?](help-compiler-option.md)|Displays a usage message to stdout.|  
-|-additionalfile|Names additional files that don't directly affect code generation but may be used by analyzers for producing errors or warnings.|  
-|[-addmodule](addmodule-compiler-option.md)|Links the specified modules into this assembly|  
-|-analyzer|Run the analyzers from this assembly (Short form: -a)|  
-|[-appconfig](appconfig-compiler-option.md)|Specifies the location of app.config at assembly binding time.|  
-|[-baseaddress](baseaddress-compiler-option.md)|Specifies the base address for the library to be built.|  
-|[-bugreport](bugreport-compiler-option.md)|Creates a 'Bug Report' file. This file will be sent together with any crash information if it is used with -errorreport:prompt or -errorreport:send.|  
-|[-checked](checked-compiler-option.md)|Causes the compiler to generate overflow checks.|  
-|-checksumalgorithm:\<alg>|Specifies the algorithm for calculating the source file checksum stored in PDB.  Supported values are: SHA1 (default) or SHA256.|  
-|[-codepage](codepage-compiler-option.md)|Specifies the codepage to use when opening source files.|  
-|[-debug](debug-compiler-option.md)|Emits debugging information.|  
-|[-define](define-compiler-option.md)|Defines conditional compilation symbols.|  
-|[-delaysign](delaysign-compiler-option.md)|Delay-signs the assembly by using only the public part of the strong name key.|  
+
+The following compiler options are sorted alphabetically. For a categorical list, see [C# Compiler Options Listed by Category](listed-by-category.md).
+
+|Option|Purpose|
+|------------|-------------|
+|[@](response-file-compiler-option.md)|Reads a response file for more options.|
+|[-?](help-compiler-option.md)|Displays a usage message to stdout.|
+|-additionalfile|Names additional files that don't directly affect code generation but may be used by analyzers for producing errors or warnings.|
+|[-addmodule](addmodule-compiler-option.md)|Links the specified modules into this assembly|
+|-analyzer|Run the analyzers from this assembly (Short form: -a)|
+|[-appconfig](appconfig-compiler-option.md)|Specifies the location of app.config at assembly binding time.|
+|[-baseaddress](baseaddress-compiler-option.md)|Specifies the base address for the library to be built.|
+|[-bugreport](bugreport-compiler-option.md)|Creates a 'Bug Report' file. This file will be sent together with any crash information if it is used with -errorreport:prompt or -errorreport:send.|
+|[-checked](checked-compiler-option.md)|Causes the compiler to generate overflow checks.|
+|-checksumalgorithm:\<alg>|Specify the algorithm for calculating the source file checksum stored in PDB.  Supported values are: SHA1 (default) or SHA256.|
```

Comment:
all the others are in 3rd person, why change here?

Reply:
I'll address this in a new PR.

I'm not seeing a change in person here. What am I missing?

Reply:
It used to say specifies and it was matching the other lines - Causes, Links, Names, Displays, etc. 

Makes sense @BillWagner?